src/item.py

```diff
@@ -182,9 +188,38 @@ def read_points_values(self, csv_path=None, idf_path=None, idd_path=None):
         """
         self.read_output_variables(csv_path)
         self.read_idf_obj_values(idf_path, idd_path)
+        self.df = DateTimeEP(self.df, self.DF_YEAR).transform()
+        self.data_processing(
+            fill_method=self.item["data_processing"]["fillna"],
+            then_dropna=self.item["data_processing"]["then_dropna"],
```

Comment:
I'm not sure if about the `then_dropna` term. I really don't have a strong suggestion, the only thing I can think about is `drop_remaining_na` or `dropna_after_fillna`.

Reply:
Maybe add a set (before and after) of data file for testing the code, and showing the before/after difference.